src/backend/gitalytics_api/common.py

```diff
@@ -46,8 +47,14 @@ async def endpoint(token: SessionKey):
             .filter(dbm.Session.id == session_id) \
             .one_or_none()
 
-    if not session:
-        raise fastapi.HTTPException(fastapi.status.HTTP_401_UNAUTHORIZED)
+        if not session:
+            # maybe redirect to /#/login ('cause seems like session expired)
```

Comment:
```suggestion
```
Kannst du mit axios einen hook einbauen, der alle 401er abfängt und im frontend auf /#/login weiterleitet?
Natürlich nicht in diesem PR, aber fändest du das für sinnvoll?